packages/esm-patient-orders-app/src/components/order-price-details.scss

```diff
@@ -0,0 +1,27 @@
+@use '@carbon/layout';
+
+.priceDetailsContainer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.priceLabel {
+  font-weight: 800;
```

Comment:
No proper font use, please refer carbon design system typography to get the correct font